### ISSUE TITLE
CockroachCloud docs should be self-contained

### ISSUE THREAD

**Issue by rmloveland** [cockroachdb/docs#5739]
tl;dr: Our Cloud docs build should include all of the things a user needs to be productive in the Cloud environment in its build without the jarring "open non-Cloud things in a new tab" experience

Currently, when you go to the [SQL page in the CockroachCloud docs](https://www.cockroachlabs.com/docs/cockroachcloud/stable/learn-cockroachdb-sql.html) and want to check out our SQL support, you are given a link to the non-cloud [SQL Statements](https://www.cockroachlabs.com/docs/v19.1/sql-statements.html) page.  The page opens in a new tab and the experience is very jarring.  You are thrown into a completely new context with no clear map of "where you are", which is confusing.  This is especially the case since you were probably not expecting a new tab to be opened in a completely different documentation set that you do not understand, much of which is not relevant to your needs.

Estimated scope of this issue is:

- Identify what docs are missing from Cloud to be truly self-contained
- Figure out how to update the Cloud build process to include those docs under the Cloud TOC
- Figure out what changes are necessary to said docs (includes, etc.) to get them into Cloud build while enabling re-use of as much as possible
- Ensure that no links point from Cloud to non-Cloud things
- ... probably many other things that I can't think of right now

**Comment by rmloveland** [comment]
cc @Amruta-Ranade @jseldess FYI

Y'all already know about the need for this work, but Amruta asked me to file an issue since (AFAICT) there isn't one for this work.

**Comment by jseldess** [comment]
Thanks, @rmloveland. This really needs to be handle in the context of an overall information architecture revamp. I'm planning to schedule an offsite soon for us and other informed/interested people to get ideas generated.

**Comment by lnhsingh** [comment]
I'm going to close this for now since we did some work to change the hierarchy. This could obviously be iterated on in the future, but will open a new issue for that later.
